examples/permissioned-voting-application/scripts/vote/result.js

```diff
@@ -0,0 +1,41 @@
+const { readGlobalStateSSC } = require("algob");
+
+async function run(runtimeEnv, deployer) {
+
+	const votingAdminAccount = deployer.accountsByName.get("voting-admin-account");
+
+	// Retreive AppInfo from checkpoints.
+  const appInfo = deployer.getSSC("permissioned-voting-approval.py", "permissioned-voting-clear.py");
+	
+	// Retreive Global State
+	const globalState = await readGlobalStateSSC(deployer, votingAdminAccount.addr, appInfo.appID);
+	console.log(globalState);
+
+	// Count votes
+	let candidatea_tally = 0, candidateb_tally = 0;
```

Comment:
in JS we don't use `_` for names. Let's rename to `candidateA` and `candidateB`

Reply:
ok